.travis.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2021 Oracle and/or its affiliates. All rights reserved.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Distribution License v. 1.0, which is available at
+# http://www.eclipse.org/org/documents/edl-v10.php.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+dist: focal
+
+sudo: required
```

Comment:
Why?

Reply:
I set it in the same way it was with mail:
https://github.com/eclipse-ee4j/mail/pull/560
There was a comment to use focal.

I have removed it, now it is using Xenial.

Reply:
I believe the comment was not about xenial vs focal but about sudo

Reply:
I removed both (sudo and dist)